Dependencies/napi/napi-jsi/include/napi/napi.h

```diff
@@ -959,6 +959,7 @@ namespace Napi {
 
     bool operator ==(const Reference<T> &other) const;
     bool operator !=(const Reference<T> &other) const;
+    operator bool() const { return !IsEmpty(); }
```

Comment:
Should this be `explicit` (requiring a cast) or left as implicit? Not sure how this is intended to be used.

Reply:
I think the intention is for it to be implicit so you can do something like `if (myRef)`

Reply:
This mimics the behavior in napi-direct: https://github.com/BabylonJS/BabylonNative/blob/master/Dependencies/napi/napi-direct/include/napi/napi.h#L1112 so that you can do what Ryan said.